Doc/whatsnew/3.10.rst

```diff
@@ -623,6 +623,10 @@ Other Language Changes
   ``__annotations__``) and have a new ``__wrapped__`` attribute.
   (Contributed by Victor Stinner in :issue:`43682`.)
 
+* Complex annotation targets (everything beside ``simple name`` targets defined
```

Comment:
I think it's better to say "Annotations for complex targets do not produce any runtime effects with PEP 563", as the targets themselves are still evaluated except for the last `__setitem__()` or `__setattr__()` call.